src/error.ts

```diff
@@ -111,7 +111,7 @@ export interface ErrorDescription extends Document {
  * @category Error
  *
  * @privateRemarks
- * CSFLE has a dependency on this error, it uses the constructor with a string argument
+ * In use encryption has a dependency on this error, it uses the constructor with a string argument
```

Comment:
This should probably refer to the module as well, unless we mean that something about QE/CSFLE specifically has a dependency on this